packages/@uppy/locales/src/fr_FR.js

```diff
@@ -34,7 +34,7 @@ fr_FR.strings = {
   dashboardWindowTitle: 'Fenêtre de téléchargeur de fichier (Appuyez sur echap pour fermer)',
   dataUploadedOfTotal: '%{complete} sur %{total}',
   done: 'Terminé',
-  dropHereOr: 'Déposer les fichiers ici ou %{browse}',
```

Comment:
@aduh95 isn't the previous one more correct?

Reply:
@Murderlon I'm not sure because there is no adapted message when only 1 file is allowed to be upload

Reply:
Yes, but that is not implemented so that is also not the case in the other locales. AFAIK we can keep this as is but I'm not native French.

Reply:
Alright so I think you can keep previous one

Reply:
Can you revert that change? And let us know if you see other missing strings in French :)

Reply:
Sure :)

Reply:
I reverted that change.
I will take a look later if I see some missing strings :)